tests/integration/backward_compatible/proxy/vector_collection_test.py

```diff
@@ -181,7 +181,21 @@ def test_backupCount_valid_values_pass(self):
             name, [IndexConfig("vector", Metric.COSINE, 3)], backup_count=2, async_backup_count=2
         )
 
-    def test_backupCount(self):
+    def test_backupCount_max_value_pass(self):
+        skip_if_client_version_older_than(self, "6.0")
+        name = random_string()
+        self.client.create_vector_collection_config(
+            name, [IndexConfig("vector", Metric.COSINE, 3)], backup_count=6
+        )
```

Comment:
Would it make sense to check whether the VC was created, maybe something like:
```python
client.get_vector_collection(vc_name).blocking()
```
That would raise an exception if the vector collection doesn't exist.
Same comment for tests that creates a VC.